src/commands/index.ts

```diff
@@ -33,7 +33,7 @@ export function initialize(context: vscode.ExtensionContext) {
   context.subscriptions.push(instrumentOperationAsVsCodeCommand("java.webview.runCommand", webviewCmdLinkHandler));
   context.subscriptions.push(vscode.commands.registerCommand("java.welcome", instrumentCommand(context, "java.welcome", showWelcomeWebview)));
   const javaFormatterSettingsEditorProvider: JavaFormatterSettingsEditorProvider = new JavaFormatterSettingsEditorProvider(context);
-  const editorOptions = { webviewOptions: {enableFindWidget: true, retainContextWhenHidden: true}, supportsMultipleEditorsPerDocument: false };
+  const editorOptions = { webviewOptions: {enableFindWidget: true, retainContextWhenHidden: true}};
```

Comment:
currently `supportsMultipleEditorsPerDocument` is only available for `CustomReadonlyEditorProvider` and `CustomEditorProvider`, so this option is removed.

Reply:
This file is used to register command, please consider move the provider registration to other place

Reply:
Can we move the provider to `extension.ts` as an exported variable? Since we need `context` to initialize the provider and use `provider.showFormatterSettingsEditor()` to register the command.